php/php.editor/src/org/netbeans/modules/php/editor/completion/PHPCompletionItem.java

```diff
@@ -770,6 +808,13 @@ public List<String> getInsertParams() {
 
         @Override
         public String getSortText() {
+            if (isFirstClassCallable) {
+                // put first-class callable syntax on the last position
+                // e.g.
+                // strlen($length)
+                // strlen(...)
+                return getName() + "99"; // NOI18N
```

Comment:
`99`? 🤔

Reply:
e.g. 
```php
function func($param1, $param2 = 2, $param3 = 3) {
}
// cc items
// func($param1) "func1"
// func($param1, $param2) "func2"
// func($param1, $param2, $pram3) "func3"
```
if we use "100" instead of "99", the order is...
func(...) func100
func($param1) "func1"
func($param1, $param2) "func2"
func($param1, $param2, $pram3) "func3"

if we use "99"...
func($param1) "func1"
func($param1, $param2) "func2"
func($param1, $param2, $pram3) "func3"
func(...) func99

I think that the priority of `func(...)` is low. So, I select "99".

Reply:
I see now, thanks!